src/components/RightSidebar/RightSidebar.vue

```diff
@@ -12,6 +12,7 @@
 		:class="'active-tab-' + activeTab"
 		:toggle-classes="{ 'chat-button-sidebar-toggle': isInCall }"
 		:toggle-attrs="isInCall ? inCallToggleAttrs : undefined"
+		force-tabs
```

Comment:
Not sure about forced tabs... IMO, looked good without navigation when there was only on tab.